rero/rero-ils-ui
# document: display cover thumbnail

* Moves thumbnail logic in 'shared' library as it's used in both admin
and public-search projects.
* Adds thumbnails in professional brief and detailed view.
* Moves 'type' field below the thumbnail in admin detailed view.
* Closes rero/rero-ils#1188.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

To display thumbnails in brief and detail views of both admin and public parts of the application.
This is a part of US1812.

## Dependencies

My PR depends on `rero-ils#<xx>`'s PR(s):

N/A

## How to test?

Checkout this PR locally.
`npm i` to install shared library
`npm run start-admin-proxy` to check admin part
`npm run start-public-search-proxy` to check public part

Check thumbnail display in both projects and both views (brief and detailed) on localhost:4200.

## Code review check list
